src/Stripe.net/Services/PaymentMethods/PaymentMethodCardOptions.cs

```diff
@@ -2,7 +2,7 @@ namespace Stripe
 {
     using Newtonsoft.Json;
 
-    public class PaymentMethodCardCreateOptions : INestedOptions
+    public class PaymentMethodCardOptions : INestedOptions
```

Comment:
For later it'd be need if we flagged params that are only on create (or vice versa) in the docstring. Not worth it but it'd be a good todo to track

Reply:
yeah I think we could likely do that in the params somehow. Right now, I'm merging the params between create and update pretty far up-stream, so we'd likely need to pass down the state from pretty early on.